code/modules/surgery/organs/augments_internal.dm

```diff
@@ -143,6 +143,46 @@
 /obj/item/organ/cyberimp/brain/anti_stun/syndicate
 	syndicate_implant = TRUE
 
+
+/obj/item/organ/cyberimp/brain/linkedsurgery
+	name = "surgical serverlink brain implant"
+	desc = "A brain implant with a bluespace technology that lets you to perform an advanced surgery through your station research server."
+	slot = ORGAN_SLOT_SURGICAL_IMPLANT
+	actions_types = list(/datum/action/item_action/update_linkedsurgery)
+	var/list/advanced_surgeries = list()
+	var/static/datum/techweb/linked_techweb
+	var/updated
```

Comment:
this shouldn't be a var

Reply:
it stores the number of surgery programs in the implant. It should be there. I'll change the name more clarely.